ipv6calc/test_ipv6calc.sh

```diff
@@ -252,6 +265,9 @@ cat <<END | grep -v '^#'
 1.2.3.r4									ipv4addr
 2.2.3.4/-1									ipv4addr
 2.2.3.4/33									ipv4addr
+1.2.3.4/									ipv4addr
+/10										ipv4addr
+./10										ipv4addr
```

Comment:
This is also somehow strange, do you really want to catch that as IPv4 address - I would see this more as an unexpected and not supported notation

Reply:
> This is also somehow strange, do you really want to catch that as IPv4 address - I would see this more as an unexpected and not supported notation

Well, it is actually so. It looks like we misunderstood each other a bit... :-)

Reply:
Sorry, yes, had overseen that the diff was in the bad-case section...